bciers/apps/registration/app/components/operations/registration/OperationInformationPage.tsx

```diff
@@ -42,25 +42,31 @@ export const createOperationInformationSchema = (
   );
 
   const oneOfOptions = purposes.map((purpose) => {
+    const isRegulatedProducts =
+      purpose !== RegistrationPurposes.ELECTRICITY_IMPORT_OPERATION &&
+      purpose !== RegistrationPurposes.POTENTIAL_REPORTING_OPERATION;
     return {
-      // required: ["regulated_products"],
+      // This works:
+      // required: [isRegulatedProducts && "regulated_products"],
```

Comment:
More details below,  yes, we should use `required: [isRegulatedProducts && "regulated_products"]`